products.js

```diff
@@ -142,21 +143,21 @@ function tunedImageFnodes() {
     // return tuningRoutine;
 }
 
-function tunedTitleFnodes() {
-    // const rules = ruleset(
-    //   //get all title tags in the inserted fixture
-    //   rule(dom('div#fixture > title'), type('titleish')),
-    //
-    //   //return image with max score
-    //   rule(type('titleish').max(), out('product-title'))
-    //
-    // );
-    //
-    // function tuningRoutine(doc) {
-    //     return rules.against(doc).get('product-title');
-    // }
-    //
-    // return tuningRoutine;
+function tunedTitleFnodes(dict) {
```

Comment:
I keep having to wrack my brain for what's in dict every time. Can we use a more descriptive name?